db/migrate/20240515093336_create_payments.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePayments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :payments do |t|
```

Comment:
config enum in migration too. see this one for ref (lines 3 & 34): https://github.com/hugostanley/toonverse_be/blob/feat/users/db/migrate/20240515051316_devise_token_auth_create_workforces.rb